context-free.md

```diff
@@ -1,22 +1,30 @@
 # Context Free
 
 ##  Who is this programming language for?
+This language is for artists who are tech savvy or programming inclined. Target markets include artists that want to dabble in tech and techies that want to dabble in art.
 
 
 ## What is easy to do in this language? Why is it easy?
+Its easy to draw shapes. There is a lot of built in infrastructure to create shapes of all kinds, sizes and colors in only a few (1 to 5) lines of code. You can do recursion relatively easily and make pictures that are made up of many shapes because of the way that you are able to manipulate the x and y coordinates.
```

Comment:
Did it bother you that they only gave you 3 base shapes to work with? I was initially irritated with the dearth of predefined shapes, though I guess the point is that you should be able to construct any shape you want using just those three shapes and some rules.